src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,72 @@
+.menu {
+  background: #d12d35;
+  z-index: 1;
+
+  &__content {
+    box-sizing: border-box;
+    height: 100vh;
+  }
+
+  &__list {
+    margin: 40px 0 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item {
+    margin-bottom: 32px;
+  }
+
+  &__link {
+    display: inline-block;
+    padding-bottom: 8px;
+    border-bottom: 1px solid transparent;
+
+    color: #fff;
+    text-decoration: none;
+    font-weight: 700;
+    font-size: 22px;
+    line-height: 22px;
+    letter-spacing: 2px;
+    text-transform: uppercase;
+    transition: border-color 0.03s;
+
+    &:hover {
+      padding-bottom: 8px;
```

Comment:
Remove unnecessary 'padding-bottom: 8px;' from the hover state as it is already defined in the '&__link' class.